lua/neogit/integrations/diffview.lua

```diff
@@ -18,23 +17,33 @@ local function get_local_diff_view(section_name, item_name, opts)
     section_name = "working"
   end
 
-  local function update_files()
+  local function update_files(current_file_path)
     local files = {}
 
+    git.repo:dispatch_refresh {
+      source = "diffview_update",
+      callback = function() end,
+    }
+
+    local repo_state = git.repo.state
+    if not repo_state then
+      return files
+    end
```

Comment:
Not sure about this - the git repo should _always_ have state at this point. How would you get here with an uninitialized repository?